nbs/core.ipynb

```diff
@@ -35,7 +35,7 @@
   {
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #566.            elif df is not None:
</pre><p>This can be just an if because the other conditions either return or raise an error</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Nixtla/neuralforecast/pull/1049/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>